src/scipp/io/hdf5.py

```diff
@@ -222,9 +222,36 @@ def read(cls, group):
         return var
 
 
+def _write_mapping(parent, mapping, override=None):
+    if override is None:
+        override = {}
```

Comment:
The code never writes anything to `override`. Is that intentional?

Reply:
Yes, this is just for making `get` work below. Could have used `if override is not None and ...` instead.